utils/collect-updates/lib/collect-packages.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const collectDependents = require("./collect-dependents");
+
+module.exports = collectPackages;
+
+function collectPackages(packages, { isCandidate = () => true, onInclude } = {}) {
```

Comment:
This felt like a smart abstraction as prerelease functionality introduced a need to collect dependents from outside of `collect-updates`. Tests were added for the new function.

Reply:
Awesome, thanks!